### ISSUE TITLE
flatpak dependency EoL

### ISSUE THREAD

**Issue by jens-st** [lwindolf/liferea#1254]
```
Name       Application ID             Version        Branch         Installation
Liferea    net.sourceforge.liferea    1.14.5         stable         system
```
```
Info: runtime org.gnome.Platform branch 42 is end-of-life, with reason:
   The GNOME 42 runtime is no longer supported as of March 21, 2023. Please ask your application developer to migrate to a supported platform.
Info: applications using this runtime:
   net.sourceforge.liferea
```

**Comment by bbhtt** [comment]
Liferea needs to port to newer webkitgtk and libsoup https://github.com/lwindolf/liferea/issues/1142

We can update the runtime only after that is done.

**Comment by lwindolf** [comment]
Will be fixed with 1.15.0 (implemented in #1225)
